serverless.yml

```diff
@@ -196,6 +204,33 @@ stepFunctions:
           defaultFail:
             Type: Fail
 
+resources:
+  Resources:
+    snsTopic:
+      Type: AWS::SNS::Topic
+      Properties:
+        DisplayName: ${self:service}-${self:provider.stage}-topic
+        TopicName: ${self:service}-${self:provider.stage}-topic
+    stateDurationAlarm:
+      Type: AWS::CloudWatch::Alarm
+      Properties:
+        AlarmName: ${self:service}-${self:provider.stage}-duration-alarm
+        AlarmDescription: Notify when duration exceeds a threshold
+        Namespace: 'AWS/States'
+        Dimensions:
+          - Name: StateMachineArn
+            Value:
+              Ref: AqtsDashcaptureDashstateDashmachineDash${self:provider.stage}
+        MetricName: ExecutionTime
```

Comment:
This is not a metric supported by `serverless-step-functions`, so I wrote my own alarm.